cpp/src/parquet/properties.h

```diff
@@ -941,6 +942,15 @@ class PARQUET_EXPORT ArrowReaderProperties {
     return coerce_int96_timestamp_unit_;
   }
 
+  /// Enable Parquet supported Arrow Extension Types.
+  ///
+  /// When enabled, Parquet will use supported Arrow ExtensionTypes in mapping to Arrow
+  /// schema. Currently only arrow::extension::json() extension type is supported. This
```

Comment:
Can we decribe it more specifically? Like arrow extension type and parquet type mapping?

Reply:
I'v updated this text a little, but perhaps we should add more?